lib/workers/repository/finalise/repository-statistics.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  RenovateConfig,
+  getConfig,
+  loadJsonFixture,
+  mockedFunction,
+} from '../../../../test/util';
+import { logger } from '../../../logger';
+import { platform } from '../../../modules/platform';
+import { runRenovateRepoStats } from './repository-statistics';
+
+jest.mock('../../../modules/platform/github/pr');
+jest.mock('../../../util/http/github');
+
+const prList = loadJsonFixture('./pr-list.json');
```

Comment:
```suggestion
const prList = Fixtures.get('./pr-list.json');
```

Reply:
👀 Not yet Can you add a `getJson` function, which simply mirrors the old?

https://github.com/renovatebot/renovate/blob/5cf9123017dff463ed48bafcd7507c90065c850a/test/util.ts#L105-L111